assignments/objects.js

```diff
@@ -49,7 +94,26 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: 'Susan',
+  age: 70
```

Comment:
Missed a coma here, but even so tehre is an issue with nesting here. What i would do is create a spearate function for speak and just add reference to it in an object like `speak: speak` and name child and grandchild objects.